R/data-storage-log-file.R

```diff
@@ -72,6 +72,9 @@ DataStorageLogFile <- R6::R6Class( # nolint object_name_linter
         values, bucket = bucket, add_username = add_username, force_params = force_params
       )
 
+      if (!is.null(values$value))
+        values$value <- as.character(values$value)
+
```

Comment:
This should resolve the issue with values having conflicting types because some inputs have numeric values while some inputs have characters, e.g. browser version. SQL does this automatically when writing and the analytics app already handles all values being string. This is also more consistent since not all shiny inputs return a numeric value, e.g. dropdowns.